platform/broadcom/docker-ptf-brcm.mk

```diff
@@ -5,4 +5,4 @@ $(DOCKER_PTF_BRCM)_PATH = $(DOCKERS_PATH)/docker-ptf-saithrift
 $(DOCKER_PTF_BRCM)_DEPENDS += $(PYTHON_SAITHRIFT)
 $(DOCKER_PTF_BRCM)_LOAD_DOCKERS += $(DOCKER_PTF)
 SONIC_DOCKER_IMAGES += $(DOCKER_PTF_BRCM)
-SONIC_STRETCH_DOCKERS += $(DOCKER_PTF_BRCM)
+SONIC_BUSTER_DOCKERS += $(DOCKER_PTF_BRCM)
```

Comment:
I don't think it is necessary to append to `SONIC_BUSTER_DOCKERS`. I don't think we reference this variable anywhere. All Dockers are considered Buster by default, unless appended to `SONIC_STRETCH_DOCKERS`. @lguohan to confirm.

Reply:
Sure. Will wait for @lguohan to confirm.

Reply:
PTF stay on STRETCH.

Reply:
@jleveque , yes you are right.

Reply:
Done.